src/test/java/org/activiti/cloud/query/ApplicationTest.java

```diff
@@ -10,7 +10,7 @@
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = QueryApplication.class)
 @DirtiesContext
-public class ApplicationTestIT {
+public class ApplicationTest {
```

Comment:
Are you ok with `ApplicationIT` as name? As it is a SpringBootTest that loads the whole Spring context it's more likely an integration test.

Reply:
Yeah it should be IT really. Think we've got this wrong in other places too but one step at a time (https://github.com/Activiti/activiti-cloud-runtime-bundle/blob/master/src/test/java/org/activiti/cloud/runtime/ApplicationTest.java)

Reply:
Had to add surefire plugin in order to do this as this project doesn't use activiti cloud parent